src/general/world_effects/WorldEffectVisuals.cs

```diff
@@ -4,6 +4,6 @@
     ///   Used as a placeholder value instead of null
     /// </summary>
     None,
-
```

Comment:
Isn't this blank line required by the formatter as there's an XML comment so None is a actually a multiline block meaning that there must be a blank line after it?